File: Algorithm/Backjun/bronze5/b9498/b9498.py
# 시험 성적
# https://www.acmicpc.net/problem/9498

inputVal = int(input().strip())

if inputVal >= 90:
    print("A")
elif inputVal >= 80:
    print("B")
elif inputVal >= 70:
    print("C")
elif inputVal >= 60:
    print("D")
else:
    print("F")

File: Algorithm/Backjun/bronze5/b9498/b9498.js
// 시험 성적
// https://www.acmicpc.net/problem/9498

const fs = require('fs');
const inputData = fs.readFileSync('/dev/stdin').toString().split('\n');
const inputVal = inputData[0].split(' ').map(Number);


function main(inputVal) {
  if (inputVal >= 90) console.log("A");
  else if (inputVal >= 80) console.log("B");
  else if (inputVal >= 70) console.log("C");
  else if (inputVal >= 60) console.log("D");
  else console.log("F");
}

main(inputVal);
